### DISCUSSION TITLE
Why users has same data?

### DISCUSSION THREAD

**Discussion by Else00** [go-shiori/shiori#1000] [Feature requests] [answered]
I added another user but it shows the same link as the other users.
I deleted the link from the second user and it get deleted also on the first user.
What i'm doing wrong?

**Comment by Monirzadeh** [comment, marked as answer]
Shiori not support complete separate user right now.
We have two type user owner and none-owner
Owner can add or edit bookmarks bit none-owner just can see bookmarks but can't modified them
